src/ov_utils.cpp

```diff
@@ -63,9 +63,12 @@ std::string getTensorMapString(const std::map<std::string, std::shared_ptr<const
 
 Status tensorClone(ov::Tensor& destinationTensor, const ov::Tensor& sourceTensor) {
     OVMS_PROFILE_FUNCTION();
+    if (sourceTensor.get_element_type() == ov::element::Type_t::string) {
+        SPDLOG_DEBUG("tensorClone for native string is unsupported");
```

Comment:
Cloning is required for passing from one DAG node to another, or when ov::Tensor exits DAG. It is not needed for now since we do not have serialization. That is scope for 2024.1